ClassWork 01_17.ipynb

```diff
@@ -0,0 +1,360 @@
+{
```

Comment:
<p>No headers and detailed description.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/edaraa2/Text-Based-Analysis/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Hello Professor,</p><p>Added another file with requested comments under the name of ClassWork 01.ipynb</p>